Collection.js

```diff
@@ -76,7 +76,9 @@ export default class Collection {
       return false;
     }
     const edvDoc = await this._getEdvDocument({id: existing.id});
-    return edvDoc.delete();
+    const docResult = await edvDoc.read();
```

Comment:
This gets the document twice -- see the line above that gets the existing doc already. We should remove that line up there and just read the doc once -- and handle any exception properly (if it doesn't exist) and then remove the doc.